github/Deployment.py

```diff
@@ -71,7 +88,7 @@ def task(self):
     @property
     def payload(self):
         """
-        :type: dict
+        :type: string
```

Comment:
This is an API break.

Reply:
If I remember correctly the payload returned already was a string, even if stated otherwise here. I'll have another look at this.

Reply:
Github just returns whatever is passed. No matter which type. I've changed the assert in create_deployment to dict and reverted my changes.